modules/imgur.py

```diff
@@ -54,16 +70,18 @@ def _gallery_info(self, hash):
             raise utils.EventsResultsError()
 
     @utils.hook("received.command.imgur", min_args=1)
-    def imgur(self, event):
+    def imgur(self, event, auto: int=0):
```

Comment:
no - separate out the bulk of the functionality in this function out in to a `_imgur` function which just takes 1 argument (`url` or so) and returns the formatted information, then call `_imgur()` from `imgur()` having pulled the url out of the `event` object

Reply:
This would work better as I could format it a lot better. I'll start working on this.

Reply:
Job's a goodun.